idd/Energy+.idd.in

```diff
@@ -34502,6 +34502,4630 @@ ZoneHVAC:EvaporativeCoolerUnit,
         \type object-list
         \object-list DesignSpecificationZoneHVACSizingName
 
+ZoneHVAC:HybridUnitaryHVAC,		
+		\memo Hybrid Unitary HVAC. A black box model for multi-mode packaged forced air equipment. Independent variables include outside air conditions and indoor air conditions. Controlled inputs include operating mode, supply air flow rate, and outside air faction. Emperical lookup tables are required to map supply air temperature supply air humidity, electricity use, fuel uses, water use, fan electricity use, and external static pressure as a function of each indpednent varaible and each controlled input. In each timestep the model will choose one or more combinations of settings for mode, supply airflow rate, outside air faction, and part runtime fraction so as to satisfy zone requests for sensible cooling, heating, ventilation, and/or dehumidification with the least resource consumption. Equipment in this class may consume electricity, water, and up to two additional fuel types.
```

Comment:
This should be marked as an extensible object, with a begin-extensible tag at the start of the extensible fields.  You could then reduce the size and maintenance burden of this huge object by not including so many.